### ISSUE TITLE
Create protocol documentation

### ISSUE THREAD

**Issue by sobomax** [sippy/rtpproxy#1]
It would be nice to have an extensive and easy to use protocol/API documentation that folks like OpenSIPS or Kamalio can use to integrate and test. I'd start with the protocol level, explaining some design decisions and constrains and go with more detailed format for each command and its options. There are some source code reorg changes that are pending on my dev box that can be useful as a starting point.

Below is the list that we can start with.

 o "U[opts] callid remote_ip remote_port from_tag [to_tag] [notify_socket notify_tag]";
 o "L[opts] callid remote_ip remote_port from_tag [to_tag]";
 o "D[w] callid from_tag [to_tag]";
 o "P[n] callid pname codecs from_tag [to_tag]";
 o "R[-xxx-] call_id from_tag [to_tag]";
 o "C[-xxx-] call_id -XXX- from_tag [to_tag]";
 o "S call_id from_tag [to_tag]";
 o "VF feature_num";
 o "V";
 o "I[b]";
 o "Q call_id from_tag [to_tag]";
 o "X";

**Comment by jevonearth** [comment]
We have two protocol docs now;

Human orientated; 
https://github.com/sippy/rtpproxy/wiki/RTPProxy-Command-Protocol

More technical, with Backus-Naur Form (BNF) definitions. from @lemenkov 
https://github.com/sippy/rtpproxy/wiki/RTPP-(RTPproxy-protocol)-technical-specification
